### ISSUE TITLE
[6.4] Unable to load Single Player game.

### ISSUE THREAD

**Issue by pamduncan** [StrangeLoopGames/EcoIssues#3438]
started an SP game.  First issue is that when netsh command has to run to set port you get the pop up to allow the change - even if running as admin.  This might bother some people or block them if they don't get get the window

SP proceeded to set up - assume it did that correctly could not find a setup log but neither did I see any crash logs.

Started to generate the world, after a good bit of time the db files were created and after 15 minutes it created a .eco file named after what I had named teh game.  CHecking it and it was an archive comatining the normally expecyed objects for  a Game.eco saved game.

Game sat at loading screen with spinning wheel for 40 more minites.  No crash or other log to give a clue what its doing.

Shut down game and backed up the test.eco saved world.  Started game back up and picked 'load; and my game 'test'

It tried to load it for another 20 minutes.  Nothing happened, world did not load but no crash log was written and game appears to still be doing something but all I get is a black screen with a  wheel turning in the middle:  [![Screenshot from Gyazo](https://gyazo.com/c9db462fc7f0dac60ae6735b33902170/raw)](https://gyazo.com/c9db462fc7f0dac60ae6735b33902170)

Attached is the game file that was generated for this sp game:  https://www.dropbox.com/s/76am9urfcupivrf/test.eco?dl=0

**Comment by metachronism** [comment]
fixed this, had not much to do with local server, had bad config dirs

**Comment by noblackthunder** [comment]
still have that issue with the released version

**Comment by pamduncan** [comment]
This is the latest  local game I have tried to generate: 

https://www.dropbox.com/s/it4deaso909qnu0/SimpleWorld_Storage.zip?dl=0

**Comment by pamduncan** [comment]
The last hotfix (9PM 1/17) seems to have fixed the issue - I can now generate a world, and see it generating on screen - not just a spinning wheel.  It gets to encasement at which point it hangs at the encasement phase.  After waiting an hour I closed out of the game and it had built a world save and the db files.  This time after closing the game down I was able to log back in and under "Continue" it showed that I had the sp game (instead of the server I had played on last) and allowed me to log in.
